source/chapters/appendix/commandline_dev_tools.rst

```diff
@@ -40,50 +40,58 @@ types, go to :ref:`file-format-compatibility`.
 
 
 
---resourcePath PATH     Top-level directory where Mixxx will look for
-                        its resource files such as MIDI mappings,
-                        overriding the default installation location.
---settingsPath PATH     Top-level directory where Mixxx will look for
-                        user settings files such as the library database
-                        and preferences configuration file.
---controllerDebug       Log all controller data Mixxx sends and receives
-                        as well as scripts it loads.
---developer             Enable developer mode. Includes extra logs, stats
-                        on performance and the Developer tools menu as
-                        well as tooltips and logs useful for skin
-                        developers.
---safeMode              Disable OpenGL widgets (scrolling waveforms,
-                        spinnies) to work around GPU driver bugs.
-                        If Mixxx is crashing on startup, try using this.
-
-                        * Automatically loads empty waveforms
-                        * Disables spinning vinyl widgets
-                        * Disables synchronization polling
-                        * Doesn't open controllers by default
---locale LOCALE         Use a custom locale for loading translations
-                        (e.g ``fr``)
--f, --fullScreen        Start Mixxx in full-screen mode.
---logLevel LEVEL        Set the verbosity of command line logging.
-
-                        ============  ==========================
-                        Value         Meaning
-                        ============  ==========================
-                        ``critical``  Critical/Fatal only
-                        ``warning``   Above + Warnings
-                        ``info``      Above + Informational messages
-                        ``debug``     Above + Debug/Developer messages
-                        ``trace``     Above + Profiling messages
-                        ============  ==========================
---logFlushLevel LEVEL   Set the logging level at which the log buffer
-                        is flushed to ``mixxx.log``.
-                        ``LEVEL`` is one of the values defined at ``--logLevel``
-                        above.
---debugAssertBreak      Breaks (``SIGINT``) Mixxx if a ``DEBUG_ASSERT`` evaluates
-                        to false. A debugger can then be used to continue.
-                        This disables the ``MIXXX_DEBUG_ASSERTIONS_FATAL``
-                        flag which can otherwise lead to a time consuming
-                        full rebuild.
--h, --help              Display this help message and exit
+--resource-path PATH            Top-level directory where Mixxx will look for
+                                its resource files such as MIDI mappings,
```

Comment:
```suggestion
                                its resource files such as skins and MIDI mappings,
```

Reply:
These are taken straight from mixxx, so we'd need to change the text there as well for consistency sake.